osu.Game/Overlays/Chat/Tabs/ChannelTabControl.cs

```diff
@@ -87,6 +87,12 @@ public void RemoveChannel(Channel channel)
         {
             RemoveItem(channel);
 
+            if (Items.Count() <= 1)
```

Comment:
without this, what does current become? that sounds like a potential bug with `TabControl` if it's not null.

Reply:
Without this, current becomes the channelselector, which this is blocking.

Reply:
This is *very* unclear. and if that's the case, why are you checking `<= 1`? why aren't you matching against the channel selector type, which is local to this class? That was it would actually be clear what you're trying to do here.

Reply:
Its kinda mirroring the logic inside `tabCloseRequested`, though that logic will likely have to be reworked anyway (considering what we need to do to fix https://github.com/ppy/osu/issues/5161)